python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -683,6 +683,27 @@ def add_nodes_from(self, nodes):
         """
         self._Impl._nodes["all_nodes"] = cudf.Series(nodes)
 
+    def density(self):
```

Comment:
We're slowly adding type annotations, starting with all new public APIs, so this will need an annotation too.

Reply:
It will be updated to def density(self) -> float:

Reply:
fixed